### DISCUSSION TITLE
Not receiving email notifications for recurring transactions

### DISCUSSION THREAD

**Discussion by mdotkurmi** [firefly-iii/firefly-iii#5951] [Firefly III]
I have docker installation of Firefly and have configured the email via .env.
I am able to send test email through Firefly administration page.
I am using home assistant automation to run the cron job every day 3:00am 'curl https://firefly.mydomain.in/api/v1/cron/[token]'

When I access the page https://firefly.mydomain.in/api/v1/cron/[token] I get the following response
_{"recurring_transactions":{"job_fired":false,"job_succeeded":false,"job_errored":false,"message":"It has been 7h after since the recurring transactions cron-job has fired. It will not fire now."},"auto_budgets":{"job_fired":false,"job_succeeded":false,"job_errored":false,"message":"It has been 7h after since the auto budget cron-job has fired. It will not fire now."}}_

Do I need to do any other setting to get the email notification and recurring transaction?

**Comment by JC5** [comment]
Your log files will tell you if and why the message was (not) sent.
